NitroxClient/NitroxClient.csproj

```diff
@@ -46,12 +48,6 @@
       <HintPath>..\packages\LiteNetLib.0.8.2-beta\lib\net35\LiteNetLib.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="UnityEngine">
```

Comment:
As mentioned in another comment, just leave the references like this, instead of the autogenerated one that adds a `Version` field etc, only to do away with it by setting `SpecificVersion` to false.